lib/app.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:client_common/models/user_application_model.dart';
+import 'package:lenra_ui_runner/lenra_application_model.dart';
+import 'package:lenra_ui_runner/lenra_ui_controller.dart';
+import 'package:lenra_ui_runner/models/channel_model.dart';
+import 'package:lenra_ui_runner/models/client_widget_model.dart';
+import 'package:lenra_ui_runner/models/socket_model.dart';
+import 'package:flutter/material.dart';
+import 'package:lenra_ui_runner/widget_model.dart';
+import 'package:provider/provider.dart';
+
+class App extends StatefulWidget {
```

Comment:
It seems that you are not using any state in the App Widget. Is it possible to use a StatelessWidget here ?